src/bidiMapper/domains/context/BrowsingContextImpl.ts

```diff
@@ -771,12 +771,12 @@ export class BrowsingContextImpl {
     switch (params.origin) {
       case 'document': {
         script = String(() => {
-          const element = document.documentElement;
+          const rect = document.body.getBoundingClientRect();
```

Comment:
Would not it make the problem worse meaning given that the body can be made of size 0x0 while the content is still visible?

Reply:
Yea, that's a good argument.